a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -764,33 +795,32 @@ def revert(
             vthunder,
             lb_resource,
             vrid_list,
+            subnet,
             *args,
             **kwargs):
-        if isinstance(result, failure.Failure):
-            LOG.exception(
-                "Unable to allocate & configure VRRP Floating IP Port")
-            return
 
-        if self.fip_port:
-            LOG.warning(
-                "Reverting VRRP floating IP delta task for vrid %s on lb_resource %s",
-                str(vrid_list),
-                lb_resource.id)
+        LOG.warning(
+            "Reverting VRRP floating IP delta task for lb_resource %s",
+            lb_resource.id)
+        # Delete newly added ports
+        for port in self.added_fip_ports:
             try:
-                self.network_driver.delete_port(self.fip_port.id)
-                # if vrid:
-                #    self.axapi_client.vrrpa.update(vrid.vrid, floating_ips=[vrid.vrid_floating_ip])
-            except req_exceptions.ConnectionError:
-                LOG.exception(
-                    "Failed to connect A10 Thunder device: %s",
-                    vthunder.ip_address)
+                self.network_driver.delete_port(port.id)
```

Comment:
Parts added while handling proper reverts story, seems to be removed here as well.  Is it alright ?

Reply:
Added try-except for delete port call